que_test.py

```diff
@@ -0,0 +1,103 @@
+import pytest
+
+from que_ import Queue
+
+
+def test_innit():
+    my_queue = Queue([1, 2, 3])
+    assert my_queue.dll.tail.value == 1
+    assert my_queue.dll.tail.prev.value == 2
+    assert my_queue.dll.head.value == 3
+    assert my_queue.dll._size == 3
```

Comment:
`_` means a method is protected and should not be used except local to where it is defined.  In this case, _size is defined on the dll so can not be used inside these que tests.  use the publicly facing len() method.

https://towardsmachinelearning.org/publicprotected-private-members-in-python/